packages/storybook/src/nimble/anchor-tabs/anchor-tabs.stories.ts

```diff
@@ -27,6 +29,49 @@ interface ToolbarArgs {
     toolbar: boolean;
 }
 
+const simpleTabs: readonly AnchorTabArgs[] = [
+    {
+        title: 'Google',
+        id: '1',
+        disabled: true,
+        href: 'https://www.google.com'
+    },
+    { title: 'NI', id: '2', disabled: false, href: 'https://www.ni.com' },
+    { title: 'Nimble', id: '3', disabled: false, href: 'https://nimble.ni.dev' }
+] as const;
+
+const wideTabs: readonly AnchorTabArgs[] = [
+    {
+        title: 'Tab 1 that is too long to fit in the drop down width',
+        id: '0',
+        disabled: false,
+        href: 'https://nimble.ni.dev'
+    },
+    {
+        title: 'Tab 2 that is also too long but disabled',
+        id: '1',
+        disabled: true,
+        href: 'https://nimble.ni.dev'
+    },
+    { title: 'Short', id: '2', disabled: false, href: 'https://nimble.ni.dev' }
+] as const;
+
+const manyTabs: AnchorTabArgs[] = [];
+for (let i = 0; i < 100; i++) {
+    manyTabs.push({
+        title: `Tab ${i}`,
+        id: `${i}`,
+        disabled: false,
+        href: 'https://nimble.ni.dev'
```

Comment:
in each of these urls that target nimble.ni.dev can you add a query parameter so it's possible to see the url is updating quickly

```suggestion
        href: `https://nimble.ni.dev?tabid=${i}`
```